contracts/src/v0.8/l2ep/test/FeedConsumer.sol

```diff
@@ -1,9 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {AggregatorV2V3Interface} from "../shared/interfaces/AggregatorV2V3Interface.sol";
+import {AggregatorV2V3Interface} from "../../shared/interfaces/AggregatorV2V3Interface.sol";
 
 contract FeedConsumer {
+  // solhint-disable-next-line
   AggregatorV2V3Interface public immutable AGGREGATOR;
```

Comment:
Seems to be just a test file, maybe we can keep the solhint warning and create a ticket for the 'l2ep' team to fix it? Shouldn't affect prod bytecode

Reply:
We don't want any active solhint warning, we can ignore it and then create a ticket with the l2ep team to fix it though. Doesn't seem very high prio imo as it's just a test file